torch/_torch_docs.py

```diff
@@ -9920,6 +9920,35 @@ def merge_dicts(*dicts):
 See also :func:`torch.t`.
 """.format(**common_args))
 
+add_docstr(torch.transpose_copy,
+           r"""
+transpose_copy(input, dim0, dim1) -> Tensor
```

Comment:
It's missing the `out` keyword argument.

Reply:
I presume that the doc-strings have no influence on https://github.com/pytorch/pytorch/pull/68867#discussion_r756385580 , so holding on adding `out=` to docs.

Reply:
Added `out=`.